app/src/main/java/net/simplifiedcoding/shelounge/DetailInterfaces/InsertFeedback.java

```diff
@@ -17,7 +17,7 @@ public void insertUser(
 
             @Field("user") String user,
             @Field("rating") float rating,
-            @Field("comment") String comment,
+            //  @Field("comment") String comment,
```

Comment:
don't comment remove if not of any use or mention todo if it would be used later